avaframe/runAna3AIMEC.py

```diff
@@ -41,12 +41,12 @@
 cfgSetup = cfg['AIMECSETUP']
 anaMod = cfgSetup['anaMod']
 
-# set parameter for standard aimec analysis with mass analysis
-cfgSetup['resType'] = 'ppr'
-cfgSetup['thresholdValue'] = '1'
-cfgSetup['diffLim'] = '5'
-cfgSetup['contourLevels'] = '1|3|5|10'
-cfg['FLAGS']['flagMass'] = 'True'
+# # set parameter for standard aimec analysis with mass analysis
+# cfgSetup['resType'] = 'ppr'
+# cfgSetup['thresholdValue'] = '1'
+# cfgSetup['diffLim'] = '5'
+# cfgSetup['contourLevels'] = '1|3|5|10'
+# cfg['FLAGS']['flagMass'] = 'True'
```

Comment:
why is this now commented?

Reply:
it took me half an hour to find out why I could not analyze my simulations with pfv and another threshold. this is because of this block. I dont know who added it but it should no be hard coded.

Reply:
just remove it